README.md

```diff
@@ -149,3 +105,11 @@ pod2g for steaks4uce exploit
 walac for pyusb
 
 checkra1n team, littlelailo for the idea of just removing the call to image4_load
+
+linushenze for ipwndfu patches
+
+hack-different team, for actively maintaining & updating ipwndfu
```

Comment:
ah yes, crediting ourselves in the README.

Reply:
if u want i can remove it..